Python/Product/IronPython/Interpreter/IronPythonInterpreter.cs

```diff
@@ -126,7 +126,8 @@ class IronPythonInterpreter :
         }
 
 #if DEBUG
-        class AssertListenerInitializer {
+        [Serializable]
+        class AssertListenerInitializer : MarshalByRefObject {
```

Comment:
I'll just make it serializable, I'm not sure how this ever worked, I just ran into the exception while debugging something else.